doc/models/Line/Parameters.md

```diff
@@ -0,0 +1,537 @@
+({cite:p}`Maxwell_1873` page 89).
+
+```{math}
+\begin{aligned}
+    \lambda_{ij}&= \dfrac{1}{2\pi\varepsilon}\ln\left(\dfrac{D'_{ij}}{D_{ij}}\right) \quad\text{if } i\neq j\\
+    \lambda_{ii}&= \dfrac{1}{2\pi\varepsilon}\ln\left(\dfrac{D'_{i}}{R_i}\right) \quad\text{otherwise}\\
+\end{aligned}
+```
+
+where:
+
+- $\varepsilon$ is the permittivity of the insulator in F/m
+- $D_{ij}$ the distance between the center of the conductor $i$ and the conductor $j$;
+- $R_i$ the radius of the conductor $i$;
+- $D'_i$ the distance between the conductor $i$ and its image with respect to the ground;
+- $D'_{ij}$ the distance between the conductor $i$ and the image of the conductor $j$ with respect to the ground.
+
+The method of images ({cite:p}`wiki:Method_Of_Image_Charges`) is depicted in the following figure. It indicates how
+to compute the distances based on the position of wires.
+
+````{tab} Planar ground
+```{image} /_static/Line/Image_Method_Plane.svg
+:alt: Image method for a planar ground
+:width: 400px
+:align: center
+```
+````
+
+````{tab} Circular ground
+```{image} /_static/Line/Image_Method_Circle.svg
+:alt: Image method for a circular ground
+:width: 400px
+:align: center
+```
+````
+
+The capacitance matrix $C$ is then defined by:
+
+```{math}
+C=
+\begin{pmatrix}
+    C_{\mathrm{a}} & C_{\mathrm{ab}} & C_{\mathrm{ac}} & C_{\mathrm{an}}\\
+    C_{\mathrm{ab}} & C_{\mathrm{b}} & C_{\mathrm{bc}} & C_{\mathrm{bn}}\\
+    C_{\mathrm{ac}} & C_{\mathrm{bc}} & C_{\mathrm{c}} & C_{\mathrm{cn}}\\
+    C_{\mathrm{an}} & C_{\mathrm{bn}} & C_{\mathrm{cn}} & C_{\mathrm{n}}
+\end{pmatrix}
+=\lambda^{-1} \quad\text{in F/km}
+```
+
+#### Conductance
+
+The conductance matrix $G$ (in S/km) is defined by:
+
+```{math}
+G=
+\begin{pmatrix}
+    G_{\mathrm{a}} & 0 & 0 & 0\\
+    0 & G_{\mathrm{b}} & 0 & 0\\
+    0 & 0 & G_{\mathrm{c}} & 0\\
+    0 & 0 & 0 & G_{\mathrm{n}}
+\end{pmatrix}
+```
+
+where $G_i=C_i\omega\tan\delta$ for all $i\in\{\mathrm{a},\mathrm{b},\mathrm{c},\mathrm{n}\}$.
+
+$\tan\delta$ is the loss tangent and is taken from this table:
+
+| Insulator                        | Loss tangent $\tan\delta$ |
+| :------------------------------- | :------------------------ |
+| PolyVinyl Chloride (PVC)         | $600\times10^{-4}$        |
+| High-Density PolyEthylene (HDPE) | $6\times10^{-4}$          |
+| Low-Density PolyEthylene (LDPE)  | $6\times10^{-4}$          |
+| Cross-linked polyethylene (PEX)  | $30\times10^{-4}$         |
+| Ethylene-Propylene Rubber (EPR)  | $125\times10^{-4}$        |
+
+Finally, the impedance matrix and the admittance matrix can be computed.
+
+```{math}
+\begin{aligned}
+    \underline{Z} &= R + j\omega L \\
+    \underline{y} &= G + j\omega C \\
+\end{aligned}
+```
+
+```{warning}
+The admittance matrix $\underline{y}$ shouldn't be confused with the shunt admittance matrix
+$\underline{Y}$ defined [here](models-line-shunt-admittance-matrix).
+```
+
+```{note}
+The frequency used to compute $\omega$ is 50 Hz.
```

Comment:
OT; shouldn't we look into allowing the user to define the frequency in RLF? This will be useful for american networks.

Reply:
Yes, it should be done in a future release.